src/PatchworkProtocol.sol

```diff
@@ -541,26 +541,28 @@ contract PatchworkProtocol {
         } else {
             revert NotAuthorized(msg.sender);
         }
+        bytes32 targetRef;
         // reduce stack to stay under limit
-        uint64 ref;
-        {
-            (uint64 _ref, bool redacted) = IPatchworkLiteRef(target).getLiteReference(fragment, fragmentTokenId);
-            ref = _ref;
-            if (ref == 0) {
-                revert FragmentUnregistered(address(fragment));
-            }
-            if (redacted) {
-                revert FragmentRedacted(address(fragment));
-            }
-            if (scope.liteRefs[ref]) {
-                revert FragmentAlreadyAssignedInScope(scopeName, address(fragment), fragmentTokenId);
-            }
+        address _target = target;
```

Comment:
@robdoesstuff remind me what the purpose is of these temp variable. is it a stack depth thing?

Reply:
Yeah EVM can only access 16 most recent variables on the stack because... reasons I guess.